.github/workflows/tests.yml

```diff
@@ -6,9 +6,6 @@ on:
       - '[0-9].*'
   pull_request:
   merge_group:
-  schedule:
```

Comment:
hi,
What is the reason of this part of code be removed? 
Because I think that is should be used for other different thing.